File: latihan b/Latihan2 C++/Produk.cpp
#include <iostream>
#include <string>

using namespace std;

class Produk
{ 
	/*kelas yang digunakan untuk mengimplementasikan Produk/
	/*atribut Produk*/
		private:
			int price = 0;			//price
			string idproduk = "";	//idProduk
	
		public:
		
		Produk(){
		/*costructor*/
		}
		
		Produk(int a, string b){
		/*costructor*/
			this->price = a;
			this->idproduk = b;
		}
	
	/*mengeset nilai price*/
	void setprice(int a)
	{
		this->price = a;
	}
	/*mengeset nilai idProduk*/
	void setidproduk(string b)
	{
		this->idproduk = b;
	}

	/*mengembalikan nilai price*/
	int getprice()
	{
		return this->price;
	}
	/*mengembalikan nilai idProduk*/
	string getidproduk()
	{
		return this->idproduk;
	}
	
	~Produk(){
		/*desstructor*/
		}
};

File: latihan b/latihan2 PHP/Produk.php
<?php

class Produk
{ 
	/*kelas yang digunakan untuk mengimplementasikan Produk/
	/*atribut Produk*/
	
		private $price = 0;		//price
		private $idproduk = "";	//idProduk
	
		public function __construct($price = 0, $idproduk = ""){
		/*costructor*/
			$this->price = $price;
			$this->idproduk = $idproduk;
		}
	
	/*mengeset nilai price*/
	public function setprice($price)
	{
		$this->price = $price;
	}
	/*mengeset nilai idProduk*/
	public function setidproduk($idproduk)
	{
		$this->idproduk = $idproduk;
	}

	/*mengembalikan nilai price*/
	public function getprice()
	{
		return $this->price;
	}
	/*mengembalikan nilai idProduk*/
	public function getidproduk()
	{
		return $this->idproduk;
	}
	
	function __destruct(){
		/*desstructor*/
		}
}
?>